content/blog/open-sourcing-workshops-on-demand-part4-managing-the-backend.md

```diff
@@ -0,0 +1,242 @@
+
+I﻿t checks a quite long list of items like:
+
+* W﻿od System compliancy: is this really a wod system? by calling out [check_system.yml](https://github.com/Workshops-on-Demand/wod-backend/blob/main/ansible/check_system.yml) playbook. 
+
+T﻿his first check includes:  
+
+* nproc hard and soft limits 
+* n﻿ofile hard and soft limits
+* Setup sysctl params 
+
+  * net.ipv4.tcp_keepalive_time, value: "1800"  
+  * kernel.threads-max, value: "4096000"  
+  * kernel.pid_max, value: "200000" 
+  * vm.max_map_count, value: "600000"
+* S﻿etup UDP and TCP firewall rules
+* Enable services:
+
+  * Fi﻿rewalld 
+  * N﻿tp 
+* S﻿tudent Management:
+
+  * Ensure limits are correct for students accounts 
+* Copy the skeleton content under /etc/skel
+* Test `.profile` file
+* Ensure vim is the default EDITOR
+* Setup `logind.conf`
+* M﻿anage `/etc/hosts` file
+* Install the pkg update script
+* Setup `crontab` for daily pkg security update
+* Deliver create/reset/setup scripts as ansible template for variable expansion
+* Install utility scripts
+* Deliver the system scripts (`cleanup-processes.sh.j2`)
+* Installation of the cleanup-processes script
+* Setup weekly cleanup processes task
+* Enable WoD service
+* Test private tasks YAML file
+* Call private tasks if available. We perform private part before users management to allow interruption of the deliver script during normal operations - waiting till end of users management can take hours for 2000 users. Potential impact: private scripts are run before users creation, so may miss some part of setup.
+* U﻿ser Management:
+
+  * Remove existing jupyterhub users
+  * Remove Linux users and their home directory
+  * Ensure dedicated students groups exist
+  * Ensure Linux students users exists with their home directory
+  * Ensure jupyterhub students users exist
+  * Setup ACL for students with jupyterhub account
+  * Setup default ACL for students with jupyterhub account
+
+A﻿ similar set of scripts exist for the different parts of the solution ([check_api-db.yml](https://github.com/Workshops-on-Demand/wod-backend/blob/main/ansible/check_api-db.yml) for api-db server, [check_frontend.yml ](https://github.com/Workshops-on-Demand/wod-backend/blob/main/ansible/check_frontend.yml)for frontend server for instance).
+
+You should have now a better understanding of the maintenance tasks associated to the backend server. Similar actions are available for the other components of the project. Checking tasks have been created for the frontend and api-db server. Having nowostly covered all the subjects related to the backend server from an infrastructure standpoint, it is high time to discuss the content part. In my next blog, I plan to describe the workshop creation process.  Time to understand how to build up some content for the jupyterhub server!
```

Comment:
Swap placement of "have" and "now" so it reads "You should now have a better..."

In the 4th sentence, there's a weird word "nowostly". I think you meant "now mostly"

JupyterHub server!